### PR TITLE
Ami and DynamoDB updates

### PR DESCRIPTION
**Pull request by apiasetskyi**
Test results and successful build https://jenkins.devaws.dataxu.net/job/automation-tools_build/132/

### CODE REVIEW

File: moto/dynamodb2/models.py [hunk 1]
```diff
@@ -505,6 +510,21 @@ def create_table(self, name, **params):
...
+
+    def list_tags_of_resource(self, table_arn):
+        required_table = None
+        for table in self.tables:
+            if self.tables[table].table_arn == table_arn:
+                required_table = self.tables[table]
+        try:
+            return required_table.tags
+        except AttributeError as e:
+            raise e
```
Comment (cchristous) [inline moto/dynamodb2/models.py]: why catch the error if you are just going to reraise it?

File: tests/test_dynamodb2/test_dynamodb.py [hunk 2]
```diff
@@ -64,3 +66,37 @@ def test_describe_missing_table():
...
+def test_list_not_found_table_tags():
+    conn = boto3.client('dynamodb',
+                        region_name='us-west-2',
+                        aws_access_key_id="ak",
+                        aws_secret_access_key="sk")
+    arn = 'DymmyArn'
+    try:
+        conn.list_tags_of_resource(ResourceArn=arn)
+    except ClientError as exception:
+        assert exception.response['Error']['Code'] == "ResourceNotFoundException"
```
Comment (cchristous) [inline tests/test_dynamodb2/test_dynamodb.py]: add new line

Comment (apiasetskyi) [inline tests/test_dynamodb2/test_dynamodb.py]: Fixed

**Comment by apiasetskyi** [comment]
As build of PR on parent repo was successful, merging changes to our master. https://travis-ci.org/spulec/moto/builds/229218752

**Pull request merged**